backend/src/main/java/com/zpi/backend/game_instance/GameInstance.java

```diff
@@ -40,5 +41,6 @@ public GameInstance(NewGameInstanceDTO newGameInstanceDTO, Game game, User owner
         this.description = newGameInstanceDTO.getDescription();
         this.isActive = true;
         this.images = new ArrayList<>();
+        this.avgRating = 0;
```

Comment:
what's that for?

Reply:
to add default value, but right, it's not nessesery.